src/views/Home/Home.js

```diff
@@ -1,18 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
+import './Home.css';
 
 import Preview from '../../components/Preview/Preview';
 import Editor from '../../components/Editor/Editor';
 
-import './Home.css';
 
+// add useState calls here for title, subtitle, font, align, and text
 export default function Home() {
-  // add useState calls here for title, subtitle, font, align, and text
+
+  const [title, setTitle] = useState('');
+  const [subtitle, setSubtitle] = useState('');
+  const [titleFont, setTitleFont] = useState('');
+  const [align, setAlign] = useState('');
+  const [text, setText] = useState('');
 
   return (
     <main>
-      {/* pass the state variables as props to the presentational components */}
-      <Preview />
-      <Editor />
+      <Preview {...{ title, subtitle, titleFont, align, text }} />
```

Comment:
Nice work Josh! It looks like we are missing the functionality of changing the font. You may want to check naming conventions where we are calling our props within the preview component. Please reach out if you need any help.